team-validator.js

```diff
@@ -215,6 +217,16 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		nature = tools.getNature(set.nature);
+		if (nature.id && !nature.exists) {
+			if (tools.gen < 3) {
+				//gen 1-2 don't have natures, just remove them
```

Comment:
Namely, the only reason you'd want to warn for a bad Nature in Gen 3+ is because the user input invalid data and you want to warn the user in case they made a mistake, instead of just silently removing the Nature.

And the only reason you'd want to warn for a Nature at all in Gen 1-2 is because the user input invalid data and you want to warn the user in case they made a mistake, instead of just silently removing the Nature.